racingcar/model/cars.go

```diff
@@ -0,0 +1,37 @@
+package model
+
+type Cars struct {
+	Cars []*Car
+}
```

Comment:
Go에서는 일급컬렉션을 이렇게 구현합니다. 구조체가 아니라 슬라이스를 그대로 타입으로 정의해요.

구조체가 아니어도 타입에 대한 메서드를 정의할 수 있기 때문이죠.

```suggestion
type Cars []*Car
```

Reply:
헉 이거는 진짜 몰랐습니다. 메서드를 구조체 내부가 아니라 외부에서 작성하는 방식이기 때문에 가능한 방법이군요! 가르침 주셔서 감사합니다. 🙏